webapp/portlet/src/main/webapp/vue-apps/profile-contact-information/components/ProfileContactEditMultiField.vue

```diff
@@ -50,6 +52,9 @@ export default {
     propertyUpdated(){
       this.$emit('propertyUpdated',this.property);
     },
+    propertyChanged(){
+      this.$emit('propertyChanged',this.property);
```

Comment:
why should we have two different functions for property changes ?

Reply:
@IlhemEssaadi Yes but we can use the same function for both events.

Reply:
> @IlhemEssaadi Yes but we can use the same function for both events.

Yes, Done!